components/post_view/post_image.jsx

```diff
@@ -8,6 +8,9 @@ import {Client4} from 'mattermost-redux/client';
 import {postListScrollChange} from 'actions/global_actions.jsx';
 
 export default class PostImageEmbed extends React.PureComponent {
+    static proxyEncodeImageUrl = (url) =>
```

Comment:
This should have curly braces around the function body since it's across two lines

Reply:
We should add that to eslint

Reply:
Nevermind, I added it and there were a lot of errors to fix.